docs/en/transform-v2/filter.md

```diff
@@ -43,6 +48,21 @@ transform {
 }
 ```
 
+Or we can delete the field named `age` by adding a `Filter` Transform with the mode field set to `DELETE` like below:
+
+```
+transform {
+  Filter {
+    source_table_name = "fake"
+    result_table_name = "fake1"
+    fields = [age]
+    mode = "DELETE"
+  }
```

Comment:
If we rename the original field named `fields` to `include_fields`, this PR will cause Seatunnel to lose backward compatibility. All users will be required to update the already existing config files; otherwise, the original config file won't work properly. I guess it's not acceptable? @hailin0

Reply:
> If we rename the original field named `fields` to `include_fields`, this PR will cause Seatunnel to lose backward compatibility. All users will be required to update the already existing config files; otherwise, the original config file won't work properly. I guess it's not acceptable? @hailin0

If we update the config option name and want to be compatible with old option names, we can use `withFallbackKeys` . Here is a example in class `MongodbConfig`:

```
    public static final Option<String> MATCH_QUERY =
            Options.key("match.query")
                    .stringType()
                    .noDefaultValue()
                    .withDescription("Mongodb's query syntax.")
                    .withFallbackKeys("matchQuery");

```

Reply:
@hailin0 @EricJoy2048  updated as suggested, PTAL again.